shopinvader/components/access_info.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Camptocamp (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+
+
+class PartnerAccess(Component):
+    """Define access rules to partner from client side."""
+
+    _name = "shopinvader.partner.access"
+    _inherit = "base.shopinvader.component"
+    _usage = "access.info"
+    _apply_on = "res.partner"
+
+    def profile(self, partner=None):
+        return {
+            "create": False,
```

Comment:
I've listed here CRUD operations, but `create` here makes little sense because you cannot create a new profile w/out registration.